docs/dev/models/dsettlement.rst

```diff
@@ -16,6 +16,32 @@ Tutorial
 --------
 You can find the D-Settlement tutorial here: :doc:`../../community/tutorial_dsettlement`
 
+Fit for settlement plate
+--------
+With a fit calculation, the match between predicted and measured settlements can be improved by automatic scaling of soil properties.
+A close fit will improve the continued prediction of final and residual settlements.
+To perform a fit for settlement plate calculation, the following settings are required:
+
+.. code-block:: python
+
+    # In the model settings, the fit option must be activated:
+    dm.set_model(is_fit_for_settlement_plate=True)
+    # or
+    DSettlementModel().datastructure.input_data.model.is_fit_for_settlement_plate = Bool.TRUE
+
+    # In the fit calculation, the fit must be activated and the vertical along which the fit is performed must be defined:
+    dm.fit_calculation.fit_vertical_number = 1
+    dm.fit_calculation.is_fit_calculation = Bool.TRUE
+
+Note that fit_vertical_number corresponds to the (zero-based) index of vertical:
+the first vertical has index 0, the second vertical index 1, etc...
+
+Make sure that the measurement file (text file) defined in DSettlementModel().datastructure.input_data.fit is present next to the input file.
```

Comment:
The measurement file does not have to be next to the input file, but then the full path must be defined (In the test I removed the folder from the path so that it would succeed on the build agents). So I think "is present." is enough.